projects/spirals-chaos/index.md

```diff
@@ -0,0 +1,126 @@
+--- 
+authors: 
+  - "Paulina Mrksic"
+authors-url: 
+  - "https://github.com/paulinalinaa"
+layout: project
+title: "spirals chaos"
+type: project
+---
+
+![](splash.png)￼
+
+
+
+# First Algorithm
+Right at the beginning of this course we had a practical part.
+We should think to ourselves how to write a good set of rules.
+
+**Inspiration**
+
+My inspiration was a pattern which I had sketched some years ago. I modified it, I also tried it to simplify and not to use too much text at the same time. In my head I had a picture and I tried to describe this as accurate as possible.
+
+**This was my set of rules**
+
+Draw a rectangle (not too broadly) and draw from the upper left corner down to the bottom right corner a diagonal line. Paint the right triangle and repeat this endlessly. While you line up the rectangles. The pencil makes no difference.
```

Comment:
Could be highlighted as quote: 

> Draw a rectangle (not too broadly) and draw from the upper left corner down to the bottom right corner a diagonal line. Paint the right triangle and repeat this endlessly. While you line up the rectangles. The pencil makes no difference.  

    
    > Draw a rectangle (not too broadly) and draw from the upper left corner down to the bottom right corner a diagonal line. Paint the right triangle and repeat this endlessly. While you line up the rectangles. The pencil makes no difference.